src/playground-file-editor.ts

```diff
@@ -4,21 +4,22 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-import {html, css, PropertyValues} from 'lit';
-import {customElement, property, query} from 'lit/decorators.js';
-import {live} from 'lit/directives/live.js';
+import { html, css, PropertyValues } from 'lit';
```

Comment:
Looks like there was a lot of auto re-formatting in the latest commits. Mind re-running prettier?

Reply:
Ahh yeah sorry, done